horace_core/sqw/@sqw/sqw.m

```diff
@@ -390,6 +390,8 @@ function  save_xye(obj,varargin)
         function npix = get.num_pixels(obj)
             npix = obj.pix_.num_pixels;
         end
+        % if sqw object has actual pixels or pixelles object
```

Comment:
```suggestion
        % if sqw object has actual pixels or pixels object
```
avoid confusion that there might be boy pixels and girl pixels

Reply:
hm -- pixel-less - no pixels are present. "Pixelles" team may higack useful name but I would not even bother thinking about them. `pixels object` seems is something different from pixelles